.github/workflows/build-and-publish.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and Publish
+
+on:
+  push:
+
+jobs:
+  build-and-publish:
+    name: Build and Publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ vars.DOCKER_ORG }}/${{ github.event.repository.name }}
+          tags: |
+            type=ref,event=branch
+            type=ref,event=tag
+            # set latest tag for master branch
```

Comment:
```suggestion
            # set latest tag for main branch
```

Reply:
We are using develop and main for newer projects. Using master upsets some people associating it with slavery.

Reply:
And the comment doesn't match the code.  This file was copied from some other project; I didn't look at it closely.